apollo-client/src/main/java/com/ctrip/framework/apollo/internals/ConfigServiceLocator.java

```diff
@@ -79,7 +79,7 @@ private boolean tryUpdateConfigServices() {
   }
 
   private void schedulePeriodicRefresh() {
-    this.m_executorService.scheduleAtFixedRate(
+    this.m_executorService.scheduleWithFixedDelay(
```

Comment:
we used scheduleAtFixedRate intentionally, to keep a fixed rate. The default interval is 5 minutes, which should be long enough.
plus, the com.ctrip.framework.apollo.internals.ConfigServiceLocator#updateConfigServices method is synchronized, it won't cause concurrency issue.

Reply:
yes, won't cause the concurrency issue, but will cause create too many threads in theory.

Reply:
the default interval is 5 minutes, and with the current retry times and timeouts, I don't think creating too many threads is possible...

Reply:
i think make this change just in case from api view, but i also accept your point. i just want use this api to clear the design purpose.